src/Repositories/Interfaces/IRepository.cs

```diff
@@ -15,5 +15,7 @@ public interface IRepository<TEntity> where TEntity : class
         
         void Remove(TEntity entity);
         void RemoveRange(IEnumerable<TEntity> entities);
+
```

Comment:
Remove this line (we usually keep the methods separated, but here we are just defining method signatures and not the actual implementation)